lab.R

```diff
@@ -18,12 +18,16 @@
 
 ## Setup
 ## **IMPORTANT**: Add all dependencies to `DESCRIPTION`
+install.packages("dplyer")
```

Comment:
I'm guessing this is just a misspelling of dplyr.  In any case, you don't want to leave active `install.packages()` calls in scripts that other people will run. 
```suggestion
# install.packages("dplyer")
```